dongle_app/js/app.js

```diff
@@ -193,4 +193,13 @@ window.onload = function() {
       FoxPlayer.close();
     }
   });
+
+  //regenerate the QR
+  var regenerateButton = document.getElementById('regenerate');
+  regenerateButton.addEventListener('click', function(){
+    asyncStorage.removeItem('dongle', function(){
```

Comment:
Before doing the `boot`, call `SimplePush.reset()`in order to regenerate endpoint as well.